guava/src/com/google/common/net/InetAddresses.java

```diff
@@ -161,7 +161,10 @@ public static boolean isInetAddress(String ipString) {
   }
 
   /** Returns {@code null} if unable to parse into a {@code byte[]}. */
-  private static byte @Nullable [] ipStringToBytes(String ipString) {
+  private static byte @Nullable [] ipStringToBytes(String ipStringParam) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 12. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.